tests/functional/pod_and_daemons/test_noobaa_db_cleartext_postgres_password.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.framework.testlib import tier2, BaseTest, bugzilla
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2274193")
```

Comment:
we dont need this redundant bugzilla markers

Reply:
why?